### PR TITLE
[TC-1458] Update the gem, and make it work with ups

### PR DESCRIPTION
**Pull request by SkayHUN**

### CODE REVIEW

**Review by AHolcsik — APPROVED** [review]

File: lib/spree/active_shipping/ups.rb [hunk 1]
```diff
@@ -0,0 +1,1318 @@
+module Spree
```
Comment (bluta) [inline lib/spree/active_shipping/ups.rb:6]: plz add a comment with where this was coming(/ is duplicated ;) ) from

**Review by bluta — CHANGES_REQUESTED** [review]

**Review by adamFoldvari — APPROVED** [review]

File: lib/spree_active_shipping/engine.rb [hunk 2]
```diff
@@ -18,10 +18,9 @@ def self.activate
 
       #Only required until following active_shipping commit is merged (add negotiated rates).
       #http://github.com/BDQ/active_shipping/commit/2f2560d53aa7264383e5a35deb7264db60eb405a
-      ActiveShipping::UPS.send(:include, Spree::ActiveShipping::UpsOverride)
 
       # Fix Canada Post "Ready to ship" package
-      ActiveShipping::CanadaPost.send(:include, Spree::ActiveShipping::CanadaPostOverride)
+      # ActiveShipping::Carriers::CanadaPost.send(:include, Spree::ActiveShipping::CanadaPostOverride)
```
Comment (erik-terracycle) [inline lib/spree_active_shipping/engine.rb:23]: @SkayHUN Why did you comment out this line?

File: Gemfile [hunk 3]
```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'active_shipping', github: 'Shopify/active_shipping'
```
Comment (storhet) [inline Gemfile]: there is no need to add this to the Gemfile

**Review by storhet — CHANGES_REQUESTED** [review]

File: lib/spree_active_shipping/engine.rb (hunk 2 above)
Comment (SkayHUN) [reply lib/spree_active_shipping/engine.rb:23]: we use only UPS, so we don't need CanadaPost right now.

**Review by storhet — APPROVED** [review]

**Review by VasiliyMaksymov — APPROVED** [review]

**Review by bluta — APPROVED** [review]

**Pull request merged**